pkg/kdump/kdump-view.jsx

```diff
@@ -396,76 +403,55 @@ export class KdumpPage extends React.Component {
-        if (this.props.stateChanging)
-            serviceWaiting = <Spinner size="md" />;
-
         let testButton;
         if (serviceRunning) {
             testButton = (
-                <Button variant="secondary" onClick={this.handleTestSettingsClick}>
+                <Button variant="secondary" isDanger onClick={this.handleTestSettingsClick}>
                     {_("Test configuration")}
                 </Button>
             );
         } else {
             const tooltip = _("Test is only available while the kdump service is running.");
             testButton = (
                 <Tooltip id="tip-test" content={tooltip}>
-                    <Button variant="secondary" isAriaDisabled>
+                    <Button variant="secondary" isDanger isAriaDisabled>
                         {_("Test configuration")}
                     </Button>
                 </Tooltip>
             );
         }
-        const tooltip_info = _("This will test the kdump configuration by crashing the kernel.");
 
-        let kdumpSwitch = (<Switch isChecked={!!serviceRunning}
-                              onChange={this.props.onSetServiceState}
-                              aria-label={_("kdump status")}
-                              isDisabled={this.props.stateChanging || !this.props.kdumpCmdlineEnabled} />);
+        let kdumpSwitch;
+        let kdumpSwitchHelper;
         if (!this.props.kdumpCmdlineEnabled) {
-            kdumpSwitch = (
-                <Tooltip content={_("crashkernel not configured in the kernel command line")} position={TooltipPosition.right}>
-                    {kdumpSwitch}
-                </Tooltip>);
+            kdumpSwitchHelper = _("Currently not supported");
+        } else {
+            kdumpSwitch = (<Switch isChecked={!!serviceRunning}
+                onChange={this.props.onSetServiceState}
+                aria-label={_("kdump status")}
+                isDisabled={this.props.stateChanging} />);
+            kdumpSwitchHelper = serviceRunning ? _("Enabled") : _("Disabled");
+        }
+
+        let alertMessage;
+        let alertDetail;
+        if (!this.props.stateChanging && this.props.kdumpStatus && this.props.kdumpStatus.installed !== undefined) {
+            if (this.props.kdumpStatus.installed) {
+                if (this.props.reservedMemory == 0) {
+                    alertMessage = _("Parameter crashkernel not configured in the kernel command line");
+                    alertDetail = _("Append a crashkernel option to the kernel command line (e.g. in /etc/default/grub) to reserve memory at boot time. Example: crashkernel=512M");
+                } else if (this.props.kdumpStatus.state == "failed") {
+                    alertMessage = (
+                        <>
+                            {_("Service has an error")}
+                            <Button variant="link" isInline className="service-link-ct-kdump" onClick={this.handleServiceDetailsClick}>{_("more details")}</Button>
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19240-20230824-135223-f118a980-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)